docs/changelog.md

```diff
@@ -2,6 +2,19 @@
 
 ## CARET
 
+### v0.5.1 <small>\_ Mar 11, 2024</small> {id = "0.5.1"}
+
+- **Update** Remove the description of response time histogram. ([caret_doc #331](https://github.com/tier4/caret_doc/pull/331))
+- **Fix** Fixed that histograms are stacked, because overlapping histograms are hard to see. ([caret_analyze #475](https://github.com/tier4/caret_analyze/pull/475))
+- **Fix** Fixed problem with ros2 caret command not working when built without --symlink-install. ([ros2caret #175](https://github.com/tier4/ros2caret/pull/175))
+- **Update** Add a dynamic type check method for the plot API. ([caret_analyze #472](https://github.com/tier4/caret_analyze/pull/472))
+- **Fix** Fixed speeding up by limiting registration to init_events because id_remapper takes a long time to load measurement data. ([caret_analyze #480](https://github.com/tier4/caret_analyze/pull/480))
```

Comment:
```suggestion
- **Fix** Speeding up by limiting registration to init_events because id_remapper takes a long time to load measurement data. ([caret_analyze #480](https://github.com/tier4/caret_analyze/pull/480))
```